$OpenBSD: patch-configure,v 1.5 2020/11/01 09:48:48 bket Exp $

The realpath command (e.g. from GNU coreutils) is not so common outside Linux
systems. Taken from
https://git.notmuchmail.org/git?p=notmuch;a=commit;h=b042a59cdf8be7ad215268eee32d4cc114d312bb

Index: configure
--- configure.orig
+++ configure
@@ -702,9 +702,9 @@ if ! pkg-config --exists zlib; then
   rm -f compat/gen_zlib_pc
 fi
 
-printf "Checking for zlib (>= 1.2.5.2)... "
+printf "Checking for zlib (>= 1.2.3)... "
 have_zlib=0
-if pkg-config --atleast-version=1.2.5.2 zlib; then
+if pkg-config --atleast-version=1.2.3 zlib; then
     printf "Yes.\n"
     have_zlib=1
     zlib_cflags=$(pkg-config --cflags zlib)
@@ -1536,7 +1536,7 @@ EOF
     if [ $WITH_PYTHON_DOCS = "1" ]; then
         echo "tags.add('WITH_PYTHON')"
     fi
-    printf "rsti_dir = '%s'\n" $(realpath emacs)
+    printf "rsti_dir = '%s'\n" "$(cd emacs && pwd -P)"
 } > sphinx.config
 
 # Finally, after everything configured, inform the user how to continue.
